pom.xml

```diff
@@ -270,7 +270,7 @@
           <docencoding>UTF-8</docencoding>
           <docfilessubdirs>true</docfilessubdirs>
           <show>package</show>
-          <source>1.5</source>
+          <source>1.${java.version}</source>
```

Comment:
Perhaps to make the whole version as a variable?

Reply:
Since it's outdated versioning scheme for JDK, I preferred leaving `1.` from main variable